### PR TITLE
Added a new quote

### PR DESCRIPTION
**Pull request by UdithaIshan**
## ✅ New Quote Checklist
- [x] The quote(s) added are in alphabetical order by author's first name and not ❌ duplicates.
- [x] The quote(s) added are about **learning a foreign language**.
- [x] The sentence attribute does not have inner quotation marks, and is ended with a period.
- [x] Copy pasted the raw JSON `quotes.json` file to [JSONLint](https://jsonlint.com/) and made sure that the modified JSON file is valid.

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -663,6 +663,10 @@
     "author": "Victor Hugo",
     "sentence": "To rescue from oblivion even a fragment of a language which men have used and which is in danger of being lost—that is to say, one of the elements, whether good or bad, which have shaped and complicated civilization—is to extend the scope of social observation and to serve civilization."
   },
+  {
+    "author": "Victor Hugo",
+    "sentence": "To learn to read is to light a fire, every syllable that is spelling out is a spark."
```
Comment (Kvaibhav01) [inline server/data/quotes.json:668]: This quote isn't acceptable. we need quotes based on "learning a foreign language" or close to this.

**Review by Kvaibhav01 — CHANGES_REQUESTED** [review]
Please update the quote as suggested.

**Comment by UdithaIshan** [comment]
@Kvaibhav01, sorry for the misunderstanding. I've changed that. please review.

**Review by kvharish — APPROVED** [review]

**Review by jayehernandez — APPROVED** [review]
Thank you for contributing to Letra ☀️, @UdithaIshan!! Appreciate your contribution to this open source project. 🥳

---

Continue supporting Letra by:
- Leaving a star ⭐️ for this GitHub repository
- Downloading the free [Chrome extension](https://chrome.google.com/webstore/detail/letra/cjodkkjokggcaeacdhjliobekbnnmoio) and leaving a review
- Upvoting the [ProductHunt page](https://www.producthunt.com/posts/letra-2)

**Pull request merged**